README.md

```diff
@@ -46,13 +46,13 @@
 </p>
 <br/>
 
-Woodpecker is a community fork of the Drone CI system.
+Woodpecker is a simple CI engine with great extensibility.
```

Comment:
We could maybe even leave out the simple as users could think its not powerful.

Reply:
well I want to have the simple in it - as we try to be simple - without cuting down on being powerfull

Reply:
what about "... simple jet powerful ..." ?

Reply:
I get your idea with "simple" though it's quite a subjective word and people might understand it negatively (which is what @anbraten might have referred to).
(Same goes for "powerful" in the end.)

Maybe "golang-based" or "efficient" might also transmit the message? In the end people care about

- speed
- resource-efficiency
- extensibility/flexibility

and having these somehow reflected in this sentence would be great 🙂

Reply:
In terms of UX it could also be intuitive?

Reply:
It's just a single sentence... I would go with "Woodpecker is a simple yet powerful CI/CD engine with great extensibility." as a good compromise. IMO we tend to overcomplicate simple problems sometimes :see_no_evil: